src/components/Molecules/Card/Card.md

```diff
@@ -17,9 +16,34 @@ import Link from '../../Atoms/Link/Link';
 </Card>;
 ```
 
+### Card row option
```

Comment:
Looking really good, mate; could we just add 2 more examples showing the other combinations?

```
smallBreakpointRowLayout={false}
mediumBreakpointRowLayout={true}
```
  
and 

```
smallBreakpointRowLayout={true}
mediumBreakpointRowLayout={false}
```
  
Whenever we're having to do tricksy overlapping matrices of styles, it's always good to see as many of the possible variations shown; the amount of times I've missed issues by not doing so has taught me the hard way 🥴

Reply:
Again, I'm sorry I didn't think of these earlier; having to REvisit code a 2nd or 3rd time is v annoying